tests/type_inference/test_compress.py

```diff
@@ -1,21 +1,26 @@
 import pytest
 
-import spox.opset.ai.onnx.v17 as op
+import spox.opset.ai.onnx.v17 as op17
+import spox.opset.ai.onnx.v18 as op18
+import spox.opset.ai.onnx.v19 as op19
+import spox.opset.ai.onnx.v20 as op20
```

Comment:
This got in with the automatic rebase. The parametrize fixture is not needed. We should be fine only using opset version 17.

```suggestion
import spox.opset.ai.onnx.v17 as op
```